src/misc/setup.cpp

```diff
@@ -1135,13 +1135,29 @@ Verbosity Config::GetStartupVerbosity() const
 	return Verbosity::High;
 }
 
+bool CommandLine::IsOption(const std::string &name, cmd_it &it) {
+	if (!(FindEntry(("/" + name).c_str(), it, false)) &&
+	    !(FindEntry(("-" + name).c_str(), it, false)) &&
+	    !(FindEntry(("--" + name).c_str(), it, false)))
+		return false;
+	return true;
+}
+
 bool CommandLine::FindExist(char const * const name,bool remove) {
 	cmd_it it;
 	if (!(FindEntry(name,it,false))) return false;
 	if (remove) cmds.erase(it);
 	return true;
 }
 
+bool CommandLine::FindExistOption(const std::string &name, bool remove) {
+	cmd_it it;
```

Comment:
Suggest setting a default value (iterators are at risk of uninitialized values too).

``` c++
auto it = cmds.end();
```

Reply:
Done.